scripts/analytics/user_summary.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import logging
+from datetime import datetime, timedelta
+from dateutil.parser import parse
+
+from modularodm import Q
+
+from website.app import init_app
+from website.models import User
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events(date):
+    """Count how many nodes exist.
+    If no date is given, include all users up until the point when the script was called.
+    If a date is given, include all users that were created up through the end of that date.
+    """
+    right_now = datetime.utcnow()
+    is_today = True if right_now.day == date.day and right_now.month == date.month else False
+    if not is_today:
+        date = date + timedelta(1)
+
+    logger.info('Gathering a count of users up until {}'.format(date.isoformat()))
+
+
+    counts = {
+        'keen': {
+            'timestamp': date.isoformat()
+        },
+        'active_users': User.find(
+            Q('is_registered', 'eq', True) &
+            Q('password', 'ne', None) &
+            Q('merged_by', 'eq', None) &
+            Q('date_disabled', 'eq', None) &
+            Q('date_confirmed', 'ne', None) &
+            Q('date_confirmed', 'lt', date)
+        ).count(),
+        'unconfirmed_users': User.find(
+            Q('date_registered', 'lt', date) &
+            Q('date_confirmed', 'eq', None)
+        ).count(),
+        'deactivated_users': User.find(
+            Q('date_disabled', 'ne', None) &
+            Q('date_disabled', 'lt', date)
+        ).count()
```

Comment:
I'd like to see `active_users`, `unconfirmed_users`, and `deactivated_users` restructured into something like:
````
{
  'status': {
    'active': ...,
    'unconfirmed': ...,
    'deactivated': ...,
  }
}
````